UX4_ValidityReliability.ipynb

```diff
@@ -307,15 +307,15 @@
     "### 1.4 Cofindence intervals for small sample sizes\n",
     "We already know that small samples are inaccurate. Instead of relying on the normal distribution, another distribution called the *t-distribution* is often used, specifically for small $n$. T-distribution is symmetric and bellshaped, but its shape is slightly different for small sample sizes. Because the shape of the t-distribution is more spread out for small samples (small degrees of freedom) it helps to compensate for the underestimation in variability that is bound to happen when we do not have enough data.\n",
     "\n",
-    "Here is the probability density of the t-distribution (red) for n=2 (1 degree of freedom) compared to the normal distribution (red):\n",
+    "Here is the probability density of the t-distribution (red) for n = 2 (1 degree of freedom) compared to the normal distribution (red):\n",
     "![](https://upload.wikimedia.org/wikipedia/commons/c/cf/T_distribution_1df_enhanced.svg \"Title\")\n",
-    "IkamusumeFan, CC BY-SA 3.0 <https://creativecommons.org/licenses/by-sa/3.0>, via Wikimedia Commons\n",
     "\n",
+    "IkamusumeFan, CC BY-SA 3.0 <https://creativecommons.org/licenses/by-sa/3.0>, via Wikimedia Commons\n",
     "\n",
-    "Therefore, if we want to make a more accurate estimation of confidence intervals we use the value of the t-distribution (instead of the normal) for the critical value (or $\\alpha$level) that we are interested in. \n",
+    "Therefore, if we want to make a more accurate estimation of confidence intervals we use the value of the t-distribution (instead of the normal) for the critical value (or $\\alpha$ level) that we are interested in. \n",
     "$$ \\bar{x}\\pm t_{(1-\\frac{\\alpha}{2})}\\frac{s}{\\sqrt{n}}$$\n",
```

Comment:
This t-distribution formula is not very well explained, and the students might get confused with it. Maybe we should explain it better or just leave it out? I think the graph where we compare normal distribution to t-distribution is clear